tutorials/metrics.md

```diff
@@ -36,14 +36,64 @@ Beside the [default golang based metrics][2], CRI-O provides the following addit
 | `crio_image_pulls_by_digest`           | `name`, `digest`, `mediatype`, `size`                                                                                                   | Counter | Bytes transferred by CRI-O image pulls by digest.                        |
 | `crio_image_pulls_by_name`             | `name`, `size`                                                                                                                          | Counter | Bytes transferred by CRI-O image pulls by name.                          |
 | `crio_image_pulls_by_name_skipped`     | `name`                                                                                                                                  | Counter | Bytes skipped by CRI-O image pulls by name.                              |
+| `crio_image_pulls_successes`           | `name`                                                                                                                                  | Counter | Successful image pulls by image name                                     |
+| `crio_image_pulls_failures`            | `name`, `error`                                                                                                                         | Counter | Failed image pulls by image name and their error category.               |
 
-\* Available CRI-O RPC's from the [gRPC API][3]: `Attach`, `ContainerStats`, `ContainerStatus`,
-`CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`,
-`ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`,
-`PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`,
-`RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`,
-`StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`,
-`UpdateContainerResources`, `UpdateRuntimeConfig`, `Version`
+- Available CRI-O RPC's from the [gRPC API][3]: `Attach`, `ContainerStats`, `ContainerStatus`,
+  `CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`,
+  `ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`,
+  `PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`,
+  `RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`,
+  `StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`,
+  `UpdateContainerResources`, `UpdateRuntimeConfig`, `Version`
+
+- Available error categories for `crio_image_pulls_failures`:
+  - `UNKNOWN`: The default label which gets applied if the error is not known
+  - `CONNECTION_REFUSED`: The local network is down or the registry refused the
+    connection
+  - `CONNECTION_TIMEOUT`: The connection timed out during the image download
+  - `404_NOT_FOUND`: The registry does not exist at the specified resource
```

Comment:
I think this should just be NOT_FOUND. It's pretty clear NOT_FOUND is a 404 error, and none of the other codes have the http code

Reply:
Yep, fixed :+1: